detection_rules/etc/non-ecs-schema.json

```diff
@@ -56,5 +56,9 @@
   },
   "logs-windows.*": {
     "powershell.file.script_block_text": "text"
+  },
+  "logs-kubernetes.*": {
+    "kubernetes.audit.objectRef.resource": "keyword",
+    "kubernetes.audit.objectRef.subresource": "keyword"
```

Comment:
You may not have many options but you should be aware that we will be reviewing the fields used in non-ecs-schema.

https://github.com/elastic/detection-rules/issues/1776

Reply:
Thank you for the heads up. Right now I don't know of an alternative but will add a link to this rule in the rest of the PRs to keep track.